### DISCUSSION TITLE
Assign variables to a user or team

### DISCUSSION THREAD

**Discussion by chapiosh** [grafana/grafana#38352] [Dashboards, Panels & Visualizations]
Make a way to assign variables to teams and users.

I'm using grafana to show stats to companies, right now, I have to copy my "master dashboard" to a new folder, then assign the permission to the company team, then assign users to the team. That's not the problem tough, what it's hard to maintain is whenever we add more stats to our "master dashboard" we need to go to update each dashboard to reflect the new data. It would be cool, if we could assign variables to teams, or users, and then use only 1 dashboard for all companies data. That way we could only modify 1 dashboard and all the companies will have a updated dashboard, and customization of the dashboard would be based on the user/team variables.

Hope this could help lots of people.

Thanks!

**Comment by allomorphy** [comment]
This feature request seems to address a fundamental requirement for enterprise deployment, unless I am missing something.

So we also want to author and maintain a single instance of a master dashboard.
This we then want to provide this as a provisioned dashbard to each org also using folders for each team.

As soon as the $__user.login  is fixed as per https://github.com/grafana/grafana/issues/23841, we will have user level dashboards with queries filtered for the data of that user.
The next step is that we will also want to have global variables that expand the members of each team.
This will allow team leaders to have a drop down populated from a templated variable containing the team members. The team leader may then select any of their team members. 

Am I missing anything that could implement this required functionality already in Grafana?

Any pointers much appreciated,

Best wishes
Eric

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82980. Feel free to continue the discussion around this there. Thank you!
